src/App.jsx

```diff
@@ -1,8 +1,10 @@
 const App = () => {
   return (
+    <div className="font-sans">
```

Comment:
Remove this div also, because we are applying font-family at body level, all fonts will get changed automatically.

Reply:
I guess the this is also not needed